react/components/molecules/keyboard/keyboard.js

```diff
@@ -99,7 +101,9 @@ export class Keyboard extends Component {
     };
 
     _specialChars = () => {
-        return [...this.props.supportedCharacters.replace(/[a-zA-Z\d:]/g, "")];
+        const specialCharsString = this.props.supportedCharacters.replace(/[a-zA-Z\d:]/g, "");
+        const stringSplitter = new GraphemeSplitter();
+        return stringSplitter.splitGraphemes(specialCharsString);

```

Comment:
This works but wouldn't it be more semantic to first split and then filter out non-special?